middleware/indexer.js

```diff
@@ -105,8 +105,8 @@ const findReceivers = withPgClient(async (ctx) => {
     // NOTE: Looks like not doing a join is much faster (not surprising, but doesn't allow for FUNCTION_CALL filtering)
     // So potential solution might be to maintain materialized view of likely tokens and query for all receivers instead
     const { rows } = await client.query(`
-        select distinct receiver_account_id from receipts
-        join action_receipt_actions on using (receipt_id)
+        select distinct receiver_account_id
+        from action_receipt_actions
         where predecessor_account_id = $1
             and action_kind = 'FUNCTION_CALL'
```

Comment:
It should be `receipt_receiver_account_id` and `receipt_predecessor_account_id`

Reply:
thx for catching untested route!

we prob not using it in wallet rn

Reply:
It should be either removed or fixed, I believe

Reply:
Fixed